_hands-on/singularity/singularity-tutorial_running-installed.md

```diff
@@ -41,13 +41,13 @@ cutadapt -h
 
 ## Example 2: Installation requiring special commands
 
-💬 `Zonation` is one example of a software that will reguire you to prefix command with "singularity_wrapper exec".
+💬 `Zonation` is one example of software that will require you to add a prefix command with "apptainer_exec".
 
 {:start="2"}
-2. So instead of e.g. `zig4 --help`, you will need to use:
+2. So instead of, e.g., `zig4 --help`, you will need to use:
     ```bash
     module load zonation
-    singularity_wrapper exec zig4 --help
+    apptainer_wrapper exec zig4 --help
     ```
 
```

Comment:
Example 2 is outdated. We have zonation installed using Tykky now. We can either remove this example or add a new example that is installed with apptainer image. Also, apptainer_wrapper/singularity_wrapper commands don't seem to work properly now.